.circleci/config.yml

```diff
@@ -243,14 +248,6 @@ jobs:
       PYTHON: "python3.8"
     <<: *build_and_test_linux_with_cov
 
-  linux-python-37:
```

Comment:
This would be for another PR, but I wouldn't be mad if we dropped all CircleCI Linux tests (it's double-covered with GitHub actions / cibuildwheel).

Reply:
We have an issue for this, #874 , so we won't forget to handle it at another time.